### PR TITLE
added code to verify username

### PR DESCRIPTION
**Pull request by rakeshmahadasa**

### CODE REVIEW

File: bookie/views/auth.py [hunk 1]
```diff
@@ -193,8 +193,13 @@ def reset(request):
         password = params.get('new_password', None)
         new_username = params.get('new_username', None)
         error = None
-
-        if not UserMgr.acceptable_password(password):
+        
+        #Check whether username already exists.
```
Comment (mitechie) [inline bookie/views/auth.py]: Make sure to have a space after the # to start a comment.

File: bookie/views/auth.py [hunk 2]
```diff
@@ -193,8 +193,13 @@ def reset(request):
...
+        if UserMgr.get(username) == None:
```
Comment (mitechie) [inline bookie/views/auth.py]: If you compare against None, then make sure to use 'is' vs ==

```
if UserMgr.get(username) is None:
```

or you I think you could just do

```
if UserMgr.get(username)
```

**Comment by mitechie** [comment]
This is very cool. A couple of small nitpicks on the syntax. 

Any chance you're up for writing a unit test to verify that this works correctly? I'd be happy to help guide you through it.

**Comment by rakeshmahadasa** [comment]
Thank you Rick for your guidance. I would love to work on writing unit test. give me a brief idea.

**Comment by mitechie** [comment]
Been superceded

**Pull request closed without merging**